### DISCUSSION TITLE
VSCode Extension

### DISCUSSION THREAD

**Discussion by Morgy93** [spatie/ray#441] [Ideas]
Hello,

if you're using this it's always a second window you need to handle.
So it would be awesome if you could have the same functionality via some extension right in Visual Studio Code.

**Comment by RhysLees** [comment]
Like this mockup? 

![FRyJ32SWQAA3kJb](https://user-images.githubusercontent.com/43909932/189527707-71825361-fbe9-4095-af3d-f43298dac5cb.jpg)

**Comment by miquelbrazil** [comment]
It would be so dope if this happened.

It's easy to arrange my browser preview, vscode, and a column for the Ray window on my ultrawide display at home (I use Divvy for my window arrangement and management). But when working from my laptop, there's a lot of back-and-forth swiping. between all three of these windows.
